pom.xml

```diff
@@ -143,105 +146,26 @@
           <artifactId>spring-boot-maven-plugin</artifactId>
           <version>${version.org.springframework.boot}</version>
         </plugin>
-        <!--
-          Disable Jacoco check inherited from kie-parent. Remove this when it's removed from kie-parent
-          (but some projects still use the check fail build).
-        -->
-        <plugin>
-          <groupId>org.jacoco</groupId>
-          <artifactId>jacoco-maven-plugin</artifactId>
-          <executions>
-            <execution>
-              <id>default-check</id>
-              <goals>
-                <goal>check</goal>
-              </goals>
-              <phase>none</phase>
-            </execution>
-          </executions>
-        </plugin>
       </plugins>
     </pluginManagement>
   </build>
 
   <profiles>
     <profile>
+      <!--
+        TODO: For moving the coverage measurement from Travis to Jenkins, add the repository name to
+        SONARCLOUD_ENABLED_REPOSITORIES in
+        https://github.com/kiegroup/kie-jenkins-scripts/blob/master/job-dsls/jobs/pr_jobs.groovy
+      -->
       <id>run-code-coverage</id>
-      <properties>
-        <jacoco.excludes>*Lexer</jacoco.excludes>
```

Comment:
@rsynek We forgot to move this to the `prepare-agent` configuration. I can see this in Travis console now:
```
java.lang.instrument.IllegalClassFormatException: Error while instrumenting org/drools/compiler/lang/DRL6Lexer.
```
It's a mystery to me that I can't see this error in vehicle routing despite the fact that they have the same Drools artifacts in their dependency trees.

Reply:
Good catch, the fix is here:
https://github.com/kiegroup/optaweb-employee-rostering/pull/371

I checked optaweb-vehicle-routing too and as you said, no sign of this error message. The coverage reported to SonarCloud is over 80%, which looks ok, I can see in the log that the `prepare-agent` goal of jacoco maven plugin was triggered. Yet, no exception about the *Lexer class.